lib/iris/tests/unit/analysis/test_PercentileAggregator.py

```diff
@@ -15,25 +15,22 @@
 from unittest import mock
 
 import numpy as np
+import dask.array as da
 
-from iris.analysis import PercentileAggregator, _percentile
+from iris.analysis import PercentileAggregator
 from iris.coords import AuxCoord, DimCoord
 from iris.cube import Cube
+from iris._lazy_data import as_lazy_data
+from iris._lazy_data import as_concrete_data
 
 
 class Test(tests.IrisTest):
     def test_init(self):
         name = "percentile"
-        call_func = _percentile
         units_func = mock.sentinel.units_func
-        lazy_func = mock.sentinel.lazy_func
-        aggregator = PercentileAggregator(
-            units_func=units_func, lazy_func=lazy_func
-        )
+        aggregator = PercentileAggregator(units_func=units_func)
         self.assertEqual(aggregator.name(), name)
-        self.assertIs(aggregator.call_func, call_func)
         self.assertIs(aggregator.units_func, units_func)
-        self.assertIs(aggregator.lazy_func, lazy_func)
```

Comment:
Removed these assertions as they failed for obvious reasons, and I'm not clear why it matters exactly what the `call_func` and `lazy_func` are (as long as the output of the function is correct).  Happy to re-jig and reinstate if someone explains their purpose.